README.md

```diff
@@ -90,19 +90,17 @@ Rand release if required, but the change must be noted in the changelog.
 Rand is built with these features enabled by default:
 
 -   `std` enables functionality dependent on the `std` lib
--   `alloc` (implied by `std`) enables functionality requiring an allocator (when using this feature in `no_std`, Rand requires Rustc version 1.36 or greater)
+-   `alloc` (implied by `std`) enables functionality requiring an allocator
+    (when using this feature in `no_std`, Rand requires Rustc version 1.36 or
+    greater)
 -   `getrandom` (implied by `std`) is an optional dependency providing the code
     behind `rngs::OsRng`
+-   `std_rng` enables inclusion of `StdRng`, `thread_rng` and `random`
+    (the latter two *also* require that `std` be enabled)
```

Comment:
Maybe add `std` as requirement for `std_rng` in Cargo.toml?

Reply:
Why? Perhaps utility is limited, but this still passes:
```
$ cargo test --no-default-features --features std_rng --tests
...
test rngs::std::test::test_stdrng_construction ... ok
...
```